draft-ietf-taps-interface.md

```diff
@@ -1049,26 +1048,47 @@ Type:
 : Enumeration
 
 Default:
-: Disabled
+: Disabled for connections created through initiate and rendezvous, Passive for listeners
 
-This property specifies whether an application wants to take advantage of
-transferring data across multiple paths between the same end hosts. Using
-multiple paths allows connections to migrate between interfaces as
-availability and performance properties change. Possible values are:
+This property specifies whether and how applications want to take advantage of
+transferring data across multiple paths between the same end hosts.
+Using multiple paths allows connections to migrate between interfaces or aggregate bandwidth
+as availability and performance properties change.
+Possible values are:
 
 Disabled:
-: The connection will not attempt using multiple paths once established
+: The connection will not allow using multiple paths once established and will disable respective functionality in case the chosen transport supports it.
 
-Handover:
-: The connection should attempt to migrate between different paths upon interface availability changes
+Active:
+: The transport system will actively tries, if a to establish additional paths once the inital connection is established if advisable by the {{multipath-objective}} chosen by the application. The desireable paths to established are controlled by the Interface Instance and Provisioning Domain Instance properties (see {{prop-interface}} and {{prop-pvd}}).
```

Comment:
```suggestion
: The connection will negotiate the use of multiple paths if the chosen transport supports this. It will actively bring up paths based on the multipath policy ({{multipath-objective}}) selected by the application, as limited by the Interface Instance and Provisioning Domain Instance properties (see {{prop-interface}} and {{prop-pvd}}).
```